doc/intl/de_GER/README.md

```diff
@@ -0,0 +1,104 @@
+<h1 align="center">
+<img src="https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" alt="Dogecoin" width="300"/>
+<br/><br/>
+Dogecoin Core [DOGE, Ð]  
+</h1>
+
+<div align="center">
+
+[![DogecoinBadge](https://img.shields.io/badge/Doge-Coin-yellow.svg)](https://dogecoin.com)
+[![MuchWow](https://img.shields.io/badge/Much-Wow-yellow.svg)](https://dogecoin.com)
+
+</div>
+
+**WICHTIG: Seit August 2024 ist der `master` Branch die primäre Integrationsverzweigung geworden und daher Instabil.
+Bevor Sie die Binärdateien selbst kompilieren, überprüfen Sie Bitte ob eine getaggte Version für ihr Betriebssystem vorhanden ist.**
+ 
+Eine internationale Dokumentation finden Sie unter [doc/intl](doc/intl/README.md).
+ 
+Dogecoin ist eine als Gemeinschaftsprojekt betriebene Kryptowährung, die von einem Shiba-Inu-Meme inspiriert wurde.
+Die Dogecoin-Core-Software ermöglicht es Jedem, einen Knotenpunkt, so genannte "Nodes" im Dogecoin-Blockchain-Netzwerk zu betreiben.
+Dogecoin verwendet das Scrypt-Hashing-Verfahren für "Proof of Work" und wurde von Bitcoin Core und andere Kryptowährungen adaptiert.
+ 
+Informationen über anfallende Standard-Transaktionsgebühren auf dem Dogecoin-Netzwerk finden Sie unter [Transaktionsgebühren](doc/fee-recommendation.md).
+
+**Website:** [dogecoin.com](https://dogecoin.com)
+ 
+## Verwendung 💻
+ 
+Um Dogecoin Core zu verwenden, sehen Sie sich Bitte die [Installations-Anleitung](INSTALL.md) und das [Einstiegstutorial](doc/getting-started.md) an.
+ 
+Die in Dogecoin Core enthaltene JSON-RPC-API ist selbstdokumentiert und kann mit dem Befehl `dogecoin-cli help` eingesehen werden. Detailliertere Informationen zu jedem Befehl finden Sie unter `dogecoin-cli help <command>`. Alternativ können Sie sich die [Bitcoin Core-Dokumentation](https://developer.bitcoin.org/reference/rpc/) ansehen, die ein ähnliches Protokoll implementiert hat, um eine navigierbare Version zu erhalten.
+ 
+### Viele Ports
+
+Dogecoin Core verwendet den Port `22556` als Standart-Port zur Kommunkation
```

Comment:
```suggestion
Dogecoin Core verwendet den Port 22556 als Standardport zur Kommunikation
```

Changed "Standart-Port" to "Standardport" and "Kommunkation" to "Kommunikation" for correct spelling.

Reply:
I made a typo in my suggestion and forgot to add the back braces. That's on me. Definitely keep it as `22556`.